src/pages/settings/Security/SecuritySettingsPage.tsx

```diff
@@ -202,6 +206,28 @@ function SecuritySettingsPage() {
         [delegators, styles, translate, personalDetails],
     );
 
+    const onUpdateDelegateRoleButtonPress = useCallback(() => {
+        if (isActingAsDelegate) {
+            setIsNoDelegateAccessMenuVisible(true);
```

Comment:
@dangrous 
Since we are not disabling the popover here and also after pressing okay, it is shown even after action taken. 

https://github.com/user-attachments/assets/0aae4ce2-e04c-4393-9542-a6881cb815ef